### DISCUSSION TITLE
Telegram

### DISCUSSION THREAD

**Discussion by Schinter** [langflow-ai/langflow#2877] [Ideas]
Hi there! Has anyone tried integrating LangFlow with a Telegram bot?
For example, so that the admin could upload documents to RAG himself, and users would get answers with corresponding results?

**Comment by Schinter** [comment]
Can anyone help with the project with RAG integration with Telegram Bot?

**Comment by deniszagitov** [comment]
I want to do the same: primarily interact with flow via telegram bot. Does anyone knows how to achieve that?

**Comment by josx** [comment]
Did you test this https://github.com/fauzaanu/langflow-bot/ ?
